core/templates/dev/head/pages/exploration_player/ExplorationPlayer.ts

```diff
@@ -62,3 +62,19 @@ require('pages/exploration_player/ConversationSkinDirective.ts');
 require('pages/exploration_player/ExplorationFooterDirective.ts');
 require('pages/exploration_player/LearnerLocalNav.ts');
 require('pages/exploration_player/LearnerViewInfo.ts');
+
+require('services/ContextService.ts');
+require('services/PageTitleService.ts');
+require('domain/exploration/ReadOnlyExplorationBackendApiService.ts');
+
+oppia.controller('ExplorationPlayer', [
+  'ContextService', 'PageTitleService', 'ReadOnlyExplorationBackendApiService',
+  function(
+      ContextService, PageTitleService, ReadOnlyExplorationBackendApiService) {
+    var explorationId = ContextService.getExplorationId();
+    ReadOnlyExplorationBackendApiService.fetchExploration(
+      explorationId, null).then(function(response) {
```

Comment:
You can actually put `explorationId, null)` on the previous line

Reply:
Done